drivers/web-worker-driver/src/test/resources/sqljs.worker.js

```diff
@@ -0,0 +1,57 @@
+import initSqlJs from "sql.js";
```

Comment:
Just for understanding: The user still needs this js file?

Reply:
Honestly, I still don't get it: your linked project also contains a js file. I am absolutely not against this, just curious.

Reply:
To use browser storage options (and just to avoid blocking the main thread), browser SQL implementations have to run in a Web Worker. The worker can use any SQL implementation it wants (e.g. sql.js, SQLite's own WASM builds, or even Postgres) and all communicating with the main thread is done via JSON messages.

The SQLDelight `WebWorkerDriver` doesn't contain any SQL implementation details. That's all left up to the actual worker script. In my tests, I used sql.js which is why it's included in there. In my other project, I used SQLite's WASM build so I include their JS scripts. Other worker script implementations would have to include their respective underlying SQL implementation's scripts.

Reply:
Okay, and because Kotlin compiler cannot create another file or supports resources in (js) klib you need this file too: https://youtrack.jetbrains.com/issue/KT-38230/Gradle-MPP-JS-Process-JS-resources-from-dependencies